src/pages/tags.js

```diff
@@ -12,20 +12,24 @@ const TagsPage = ({
 data, location
 }) => (
   <Layout data={data} location={location}>
+  <section className="section">
   <div className="container content">
     <div>
-      <h1>Tags</h1>
+      <h1 className="title">Tags</h1>
       <ul>
```

Comment:
Maybe i can organize the tags not in unordered list but into a div:

```html
<div class="tags">
  <span class="tag">One</span>
  <span class="tag">Two</span>
  <span class="tag">Three</span>
</div>
```

see here: https://bulma.io/documentation/elements/tag/#list-of-tags